pkg/sinks/sink.go

```diff
@@ -87,7 +92,7 @@ func (u *SinkProcessor) Start(ctx context.Context) error {
 		// create reader for each partition. Each partition is a group in redis
 		for index, bufferPartition := range u.VertexInstance.Vertex.OwnedBuffers() {
 			fromGroup := bufferPartition + "-group"
-			consumer := fmt.Sprintf("%s-%v", u.VertexInstance.Vertex.Name, u.VertexInstance.Replica)
+			consumer := fmt.Sprintf("%s-%v", vertexName, u.VertexInstance.Replica)
```

Comment:
This is risky. `vertex.spec.Name` is not unique in one namespace, while `vertex.Name` is.